osu.Game.Rulesets.Osu/Objects/Drawables/DrawableHitCircle.cs

```diff
@@ -10,16 +10,21 @@
 using osu.Framework.Input.Bindings;
 using osu.Game.Rulesets.Objects.Drawables;
 using osu.Game.Rulesets.Osu.Objects.Drawables.Pieces;
+using osu.Game.Rulesets.Osu.Skinning;
 using osu.Game.Rulesets.Scoring;
-using osuTK;
 using osu.Game.Skinning;
+using osuTK;
 
 namespace osu.Game.Rulesets.Osu.Objects.Drawables
 {
     public class DrawableHitCircle : DrawableOsuHitObject, IDrawableHitObjectWithProxiedApproach
     {
         public ApproachCircle ApproachCircle { get; }
 
+        public IBindable<bool> ExpandNumberPiece => expandNumberPiece;
+
+        private readonly BindableBool expandNumberPiece = new BindableBool();
```

Comment:
Why have this in DHC if it's not being used here? MainCirclePiece/LegacyMainCirclePiece already have access to the skin.

Reply:
Right 🤦‍♂️